pkg/controller/provider/validation.go

```diff
@@ -202,10 +205,25 @@ func (r *Reconciler) validateSecret(provider *api.Provider) (secret *core.Secret
 			"thumbprint",
 		}
 	case api.OVirt:
-		keyList = []string{
-			"user",
-			"password",
-			"cacert",
+		if ovirt.GetInsecureSkipVerifyFlag(secret) {
+			provider.Status.SetCondition(libcnd.Condition{
+				Type:     ConnectionInsecure,
+				Status:   True,
+				Reason:   InsecureSkipVerify,
+				Category: Warn,
+				Message:  "TLS is susceptible to machine-in-the-middle attacks unless custom verification is used.",
```

Comment:
Maybe "TLS is susceptible to machine-in-the-middle attacks when certificate verification is skipped."

Reply:
makes sense to me too, changed